pkg/controllers/vdb/restart_reconciler.go

```diff
@@ -480,9 +480,20 @@ func (r *RestartReconciler) restartCluster(ctx context.Context, downPods []*PodF
 			startdb.WithConfigurationParams(r.ConfigurationParams.GetMap()),
 		)
 	}
+	hostsInOneSandbox := true
+	firstPodSandbox := ""
+	if len(downPods) > 0 && downPods[0].sandbox != vapi.MainCluster {
+		firstPodSandbox = downPods[0].sandbox
+	} else {
+		hostsInOneSandbox = false
+	}
```

Comment:
`downPods` here should be assumed non-empty. All checks should be done before calling this function. It will also simplify the code in this block.